static/js/popover_menus_data.js

```diff
@@ -89,6 +91,9 @@ export function get_actions_popover_content_context(message_id) {
     const should_display_read_receipts_option =
         page_params.realm_enable_read_receipts && not_spectator;
 
+    const narrow_width = message_viewport.width() < media_breakpoints_num.sm;
```

Comment:
I think we should call this variable `narrow_message_feed` -- since it's not about the width of the whole window (we have breakpoints where the sidebars hide that impact this calculation, and the math is correct, but the name should communicate clearly what we're checking).

Reply:
Hmm, though thinking about this more, wouldn't a better check be to just look at whether there's in fact a visible add reaction button in the DOM inside the message? If there isn't, we include it in the popover.

That way, we don't have to think about keeping these two implementations in sync.

Reply:
Yeah, makes sense. Added that change.